bazel/spec-bundling/test/async-await.spec.ts

```diff
@@ -0,0 +1,56 @@
+import 'zone.js';
+import 'zone.js/testing';
+
+import {JSDOM} from 'jsdom';
+import {Component} from '@angular/core';
+import {fakeAsync, flush, TestBed, waitForAsync} from '@angular/core/testing';
+import {
+  BrowserDynamicTestingModule,
+  platformBrowserDynamicTesting,
+} from '@angular/platform-browser-dynamic/testing';
+
+describe('native async/await downleveled', () => {
+  beforeAll(() => {
+    const {window} = new JSDOM();
+
+    (global as any).window = window;
+    (global as any).document = window.document;
+    (global as any).Node = window.Node;
+    (global as any).MouseEvent = window.MouseEvent;
+
+    TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+  });
+
+  it('should properly detect changes', fakeAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [AppComponent],
+    });
+    const fixture = TestBed.createComponent(AppComponent);
+    const el = fixture.nativeElement as HTMLElement;
+
+    fixture.detectChanges();
```

Comment:
Optional: Can you put in comments on why we need to detect changes and flush in this specific order?

Reply:
Done.